src/main/java/org/jitsi/jicofo/bridge/Bridge.java

```diff
@@ -72,6 +78,13 @@
      */
     private int lastReportedPacketRatePps = 0;
 
+    /**
+     * The last report stress level
+     */
+    private double lastReportedStressLevel = 0.0;
+
+    private boolean usePacketRateStat = false;
```

Comment:
I'd name this `usePacketRateStatForStress`, and comment that it's for supporting older bridge versions.

Reply:
It's also never set to `true`. Do you want to initialize it to `true`, or set to `true` every time we receive stats without the `stress` value?

Reply:
I cleaned it up a bit, take a look now.